modules/tinymce/src/themes/silver/test/ts/browser/editor/sizing/ResizeTest.ts

```diff
@@ -11,13 +11,14 @@ UnitTest.asynctest('Editor resize test', (success, failure) => {
   Theme();
 
   TinyLoader.setup((editor: Editor, onSuccess, onFailure) => {
-    const cAssertEditorSize = (expectedWidth: number, expectedHeight: number) => Chain.control(
-      Chain.op((container: SugarElement) => {
-        Assertions.assertEq(`Editor should be ${expectedHeight}px high`, expectedHeight, container.dom.offsetHeight);
-        Assertions.assertEq(`Editor should be ${expectedWidth}px wide`, expectedWidth, container.dom.offsetWidth);
-      }),
-      Guard.addLogging('Ensure that the editor has resized')
-    );
+    const cAssertEditorSize = (expectedWidth: number, expectedHeight: number) =>
+      Chain.control(
```

Comment:
Just curious if you added the newline here or if your IDE does this automatically or something?

Reply:
Yeah, I'd be curious to know why this was changed as well, as you haven't changed anything else.

Reply:
I did it for readability for me, and committed it without meaning to.